build.sbt

```diff
@@ -4,8 +4,12 @@ val scalajs = Build.root
 val ir = Build.irProject
 val irJS = Build.irProjectJS
 val compiler = Build.compiler
-val tools = Build.tools
-val toolsJS = Build.toolsJS
+val io = Build.io
+val ioJS = Build.ioJS
+val logging = Build.logging
+val loggingJS = Build.loggingJS
```

Comment:
Maybe we should just have a `base` artifact with packages io, logging and utils (where `URIUtils` and `escapeJS` go)?

Reply:
In that case, `ir` needs to depend on `base` (for `escapeJS`), and that means it depends on all of `io` and `logging`, but it really shouldn't.

(also `compiler` needs to source in `ir/src/` and all its dependencies, so that would mean sourcing in all of `io` and `logging` for no reason)